File: HappyDog.DataTransferObjects/Category/CategoryDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace HappyDog.DataTransferObjects.Category
{
    public class CategoryDTO
    {
        public int Id { get; set; }
        public string Label { get; set; }
        public string Value { get; set; }
        public string Color { get; set; }
    }
}

File: HappyDog.Java/src/main/java/wang/doghappy/java/module/category/model/CategoryDto.java
package wang.doghappy.java.module.category.model;

public class CategoryDto {
    private int id;
    private String label;
    private String value;
    private String color;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    @Override
    public String toString() {
        return "CategoryDto{" +
                "id=" + id +
                ", label='" + label + '\'' +
                ", value='" + value + '\'' +
                ", color='" + color + '\'' +
                '}';
    }
}
